lib/osc.dart

```diff
@@ -29,26 +29,21 @@ class Conn {
 
   Future<Message> receive(Duration timeout) async {
     // Create a disposable receiver using the same port as the sender's port
-    final receiver = await UDP.bind(Endpoint.any(port: sender.local.port));
-    var msg = Message();
     try {
-      await for (final event in receiver.asStream(timeout: timeout)) {
+      await for (final event in _sender.asStream(timeout: timeout)) {
```

Comment:
When using the ```asStream``` method and providing a timeout, the instance becomes unusable after the timeout.   Is there a way to set up a timeout another way?

> "Optionally, a timeout can be specified. if it is, the [UDP](https://pub.dev/documentation/udp/latest/udp/UDP-class.html) instance stops listening after the duration has passed. The instance is closed too, making the instance unusable after the timeout runs out."
[UDP asStream method](https://pub.dev/documentation/udp/latest/udp/UDP/asStream.html)

Reply:
i made the timeout completely optional now.